### DISCUSSION TITLE
Test Scripts and Project Size

### DISCUSSION THREAD

**Discussion by imdeepmind** [js-thing/http-status-codes#98] [Ideas]
Is there any specific reason not to put the test scripts in a separate folder?

I think because of this, NPM and PackagePhobia both showing the package size as 300KB, which is incredibly high considering this package. Further, because of this, some people might not use this library and move to some other library.

NPM:
![image](https://user-images.githubusercontent.com/34741145/119540175-c1e48100-bdaa-11eb-9d56-3041c9f13bea.png)

PackagePhobia Link: https://packagephobia.com/result?p=%40js-thing%2Fhttp-status-codes

Note: Once the project is bundled, the package size is incredibly low (~12KB).

**Comment by imdeepmind** [comment]
@j-a-k-git

**Reply by j-a-k-git** [reply]
@imdeepmind Good Point !
However the reason for the package being 300K is different.

If you see the node_modules folder after npm install there are **no test scripts included**
you get three subfolders folders inside node_modules/@js-thing/http-status-codes

**cjs -- 108K
esm -- 100K
types -- 82K**

cjs & esm are the same but one is compiled for **commonJS** module and **esm** for es module imports.
and types is simply for development purpose.

In any given project only one folder is in use depending on the import statement.
I believe the build tool one uses will remove the others while building.

**But i would like to enlightened if there are other ways to deal with this**

Thanks
